modules/Sources/Features/BalanceBreakdown/BalanceBreakdownView.swift

```diff
@@ -209,22 +223,28 @@ extension BalanceBreakdownView {
             Text(L10n.Balances.fee(ZatoshiStringRepresentation.feeFormat))
                 .font(.custom(FontFamily.Inter.semiBold.name, size: 11))
         }
-        .frame(height: 166)
-        .padding(.horizontal, 30)
-        .background {
-            Asset.Colors.shade92.color
+    }
+
+    @ViewBuilder private func transparentBlockHintBox(_ viewStore: BalanceBreakdownViewStore) -> some View {
+        VStack {
+            Text(L10n.Balances.HintBox.message)
+                .font(.custom(FontFamily.Inter.medium.name, size: 11))
```

Comment:
I see that the implementation uses the `medium` typeface, although the design says `normal`. The result then is slightly different.

Reply:
good catch, fixed